Engine/TransactionHandlers/BrokerageTransactionHandler.cs

```diff
@@ -757,7 +832,8 @@ private OrderResponse HandleUpdateOrderRequest(UpdateOrderRequest request)
         {
             Order order;
             OrderTicket ticket;
-            if (!_orders.TryGetValue(request.OrderId, out order) || !_orderTickets.TryGetValue(request.OrderId, out ticket))
+            if (!(_openOrders.TryGetValue(request.OrderId, out order) || _closedOrders.TryGetValue(request.OrderId, out order))
+                || !(_openOrderTickets.TryGetValue(request.OrderId, out ticket) || _closedOrderTickets.TryGetValue(request.OrderId, out ticket)))
```

Comment:
I see this condition in at least two places (and it doesn't make for easy reading) -- thoughts on adding something like:
```
bool TryGetOrderAndTicketById(int orderId, out Order order, out OrderTicket ticket)
```

Reply:
Yes, I'll add it :)